TempleApp/src/main/java/example/TempleApp/Activities/ReadSingleData.java

```diff
@@ -102,27 +102,27 @@ protected void onPreExecute() {
             super.onPreExecute();
 
             dialog = new ProgressDialog(ReadSingleData.this);
-            dialog.setTitle("Hey Wait Please...");
-            dialog.setMessage("Fetching Your Values");
+            dialog.setTitle(getString(R.string.wait));
+            dialog.setMessage(getString(R.string.fetch));
             dialog.show();
 
         }
 
         @Nullable
         @Override
         protected Void doInBackground(Void... params) {
-            Log.i(Controller.TAG, "IDVALUE" + id);
+            Log.i(Controller.TAG, getString(R.string.ID_value) + id);
             JSONObject jsonObject = Controller.readData(id);
-            Log.i(Controller.TAG, "Json obj " + jsonObject);
+            Log.i(Controller.TAG, getString(R.string.json_obj) + jsonObject);
 
             try {
                 /*
                   Check Whether Its NULL???
                  */
                 if (jsonObject != null) {
 
-                    JSONObject user = jsonObject.getJSONObject("user");
-                    name = user.getString("name");
+                    JSONObject user = jsonObject.getJSONObject(getString(R.string.user));
+                    name = user.getString(getString(R.string.name));
```

Comment:
here it is `NAME` not `NAME:` please change that

Reply:
Made corrections